images/manageiq-base/Dockerfile

```diff
@@ -69,17 +70,16 @@ COPY --from=appliance_build build/kickstarts/partials/post/python_modules.ks.erb
 RUN bash /tmp/python_modules && \
     rm -f /tmp/python_modules && \
     rm -rf /root/.cache/pip && \
-    dnf clean all && \
-    rm -rf /var/cache/dnf
+    clean_dnf_rpm
+
+# Build the RPM manifest
```

Comment:
I thought we were keeping this as close as possible to the end in case any other steps were added with `dnf` commands

Reply:
I was actually considering moving the dumb-init and ADD calls _up_ the file.  If you're ok with that for this PR I can do that, and then this is at the "end" of the file again.

Reply:
Even so, the only calls afterwards are an ADD, and the curl of dumb-init, so this is technically at the "end" of any dnf commands

Reply:
I'm fine with moving them up, they're only used at runtime, not build time so the order doesn't matter.

Reply:
Not for this PR, but dumb-init is up to v1.2.5 now, so we should probably update.  Also, should we rely on pulling it from Github at every build or do you think we should keep a copy somewhere else (package it in an RPM)?

Reply:
Oh yes I saw that - will have a follow-up PR

Reply:
@bdunne Updated.  I moved up both commands